.github/workflows/filter_testsuite.yml

```diff
@@ -0,0 +1,72 @@
+name: Filter Testsuite
+
+on:
+  issue_comment:
+      types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        if: github.event.comment.user.login == 'maxGimeno' && contains(github.event.comment.body, '/testme')
```

Comment:
Here you use [`jobs.<job_id>.steps[*].if`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif), and the conditional could be factorized to the job, using [`jobs.<job_id>.if`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif), I think. I have not tested. I just read the doc.